src/undomanager.js

```diff
@@ -22,16 +22,21 @@ define([
         $('.fd-undo-container').append(UNDO_ALERT);
     }
 
-    function toggleAlert(undoStack) {
+    function toggleAlert(undoStack, vellum) {
```

Comment:
Agreed that passing `vellum` through here does seem a little strange. Could change it to passing a generic `callback` function and have that function defined in `UndoManager`. Would be nice to also have this managed on a per form basis, but I don't think it's possible to do that with UI code like this